src/client/components/measures-tile/measures-tile.scss

```diff
@@ -18,32 +18,31 @@
 @import '../../imports';
 
 .measures-tile {
-  .row {
-    padding-left: $padding-compact;
-    height: $measure-height;
-    cursor: pointer;
-    overflow: hidden;
-    white-space: nowrap;
+  $icon-width: 37px;
 
-    &:last-child {
-      margin-bottom: 12px;
-    }
+  .rows {
+    .row {
```

Comment:
Are such complicated rules needed? This will end up as .measures-tile .rows .row - perhaps some better naming (namespacing) could allow `.row` elevation.
More complex selectors = bigger CSS specificity = harder overwriting in the future.